src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -967,6 +1462,40 @@ public function count()
         return count($this->defaults);
     }
 
+    /**
+     * Returns the nesting level.
+     *
+     * @return string[]
+     */
+    public function getNestedLevelsName(): array
+    {
+        return $this->nestedLevelsName;
+    }
+
+    /**
+     * @param array $nestedLevelsName
+     *
+     * @return self
```

Comment:
docblock to be removed (we don't as them when they don't add anything over the signature)